Nuance not supported list with NVR option (#26909)
home-assistant/home-assistant.github.io: source/_integrations/reolink.markdown

Battery-powered cameras are not yet supported.

> The following models are lacking the HTTP webserver API and can therfore not work directly with this integration.
> However these cameras can work with this integration through a NVR, in which the NVR is connected to Home Assistant.

Reviewer: ```suggestion
The following models are lacking the HTTP web server API and can, therefore, not work directly with this integration.
However, these cameras can work with this integration through an NVR in which the NVR is connected to Home Assistant.
```
